### DISCUSSION TITLE
Replace & Play Bug

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1334] [Fixed Issues]
@rern
Library options "Select Track = Replace + Play" and "Switch to Playback" are both set.

Play a Radio Paradise stream. (Other sources do not have the bug.)
Go to library and select Replace/Play for an album stored on USB.
Album Data and cover art appears, and switches to play.
Play immediately stops.
Click play, and play starts again as normal.

Same thing happens when selecting a NAS folder.

**Comment by R101G** [comment]
@rern 
Problem is still present after 20230609 update.
If I reboot it works OK once, then reverts to replace only function.

(Also, the webradio icon shows a count figure in the library when count is not enabled.)

**Comment by R101G** [comment]
@rern
Bug still present after 20230616 update.

**Reply by rern** [reply]
Try:
`/srv/http/bash/cmd.sh` line 11 - increase `sleep 2` by 1 at a time

**Reply by R101G** [reply]
Thanks @rern. Increasing to 3 works, but sometimes it does a start-stop-start. Setting to 4 is slow, but cleaner.

**Reply by rern** [reply]
Please try the UPDATE branch.

**Reply by R101G** [reply]
@rern 
Yes, it's back to normal now - thanks. (I have also added turbo mode to the config in case the speed of my Pi3B rev 1.2 is a problem.)
